website/docs/tutorials/08-interacting-with-starknet/03-deploy-account.md

```diff
@@ -0,0 +1,17 @@
+# Creating account
+
+## Overview
+
+In order to create a new account, you need to deploy an account contract. StarkNet expects a certain interface from an account contract. [OpenZeppelin](https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/openzeppelin/account/presets/Account.cairo) and [ArgentX](https://github.com/argentlabs/argent-contracts-starknet/blob/develop/contracts/account/ArgentAccount.cairo) provide account contract implementations that you can use.
+
+There are two ways of deploying a new account contract:
```

Comment:
Aren't wallets (Argent, Braavos) deploying accounts?

Reply:
I think they use DEPLOY transactions. I'll mention the third, deprecated way in the doc.

Reply:
I think they definitely shifted to deploy_account flow

Reply:
I created an ArgentX account on last Friday. They used DEPLOY tx. They couldn't create an account using the DEPLOY_ACCOUNT tx without prefunding it.

![Screen Shot 2022-11-07 at 14 16 29](https://user-images.githubusercontent.com/9629577/200319924-6fac2dbb-4f5a-4dba-a247-b5d8d8fe7151.png)

Reply:
I created an account recently with ArgentX and `DEPLOY` tx was used. I'd have to prefund the account to make the `DEPLOY_ACCOUNT` tx accepted.

There's ~20 DEPLOY_ACCOUNT txs on the testnet.
https://testnet.starkscan.co/txs